mantidproject/mantid
# IndirectCorrections - post plot and save

The plotting and saving of the results workspace has been changed to after the execution of the algorithm for all IndirectCorrections tabs.

**To test:**
Files to test with: \olympic\babylon5\Public\Louise McCann\Corrections
For each tab load relevant files and run algorithm. Check each plot option plots a suitable graph and that the appropriate workspaces have been saved in your save directory.
- [ ] Container Subtraction
- [ ] Calculate Paalman Pings
- [ ] Apply Paalman Pings
- [ ] Absorption

Fixes #17021 

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
